.github/workflows/test.yaml

```diff
@@ -1,5 +1,8 @@
 name: Test
 
+env:
+    WEB3_INFURA_PROJECT_ID: ${{ secrets.WEB3_INFURA_PROJECT_ID }}
```

Comment:
Secrets aren't shared with PR from other repos. We had to harcode the value on the mix-strategy project.

Reply:
Removed this